src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(a) {
```

Comment:
Consider using arrow functions for methods inside objects when you don't need to use their own `this` context. However, in this case, since you are using `this` to refer to the object's `result` property, the function shorthand is appropriate.